docs/dashboard_widget_field_rules.md

```diff
@@ -0,0 +1,81 @@
+# Rules for Adding Fields to Dashboard Widgets
+
+## Before Making Changes
+1. Check if the field is already implemented in the widget's schema and build functions
+2. Review the dashboard.yml API spec to understand the field's structure and requirements
+3. Look at similar implementations in other widgets for consistent patterns
+
+## Implementation Steps
+1. Start Small: Make changes one at a time and review each change independently
+2. Follow Existing Patterns: Use the same field implementation pattern as seen in other widgets
+3. Required Components:
+   - Add field to widget's schema definition function (e.g., `getTreemapDefinitionSchema`)
```

Comment:
```suggestion
   - Add field to widget's schema definition function (such as `getTreemapDefinitionSchema`)
```
Replace all instances of "e.g." with something like "such as" or "for example,", since we try to avoid e.g. in our copy.